openedx/core/djangoapps/theming/models.py

```diff
@@ -17,3 +18,21 @@ class SiteTheme(models.Model):
 
     def __unicode__(self):
         return self.theme_dir_name
+
+    @staticmethod
+    def get_theme(site):
+        """
+        Get SiteTheme object for given site, returns default site theme if it can not
+        find a theme for the given site and `DEFAULT_SITE_THEME` setting has a proper value.
+
+        Args:
+            site (django.contrib.sites.models.Site): site object related to the current site.
+
+        Returns:
+            SiteTheme object for given site or a default site set by `DEFAULT_SITE_THEME`
+        """
+
+        theme = site.themes.first()
```

Comment:
This seems like the perfect place to insert the ability to provide a theme override through URL parameter. 

https://openedx.atlassian.net/browse/FEDX-31

It would be incredibly useful to be able to dynamically switch themes on a sandbox when verifying changes. I'm thinking this would be equivalent to the current ``preview_lang`` URL parameter for dynamically switching locale.